src/cascadia/TerminalControl/init.cpp

```diff
@@ -5,6 +5,9 @@
 #include <LibraryResources.h>
 #include <WilErrorReporting.h>
 
+// For g_hCTerminalCoreProvider
+#include "../../cascadia/TerminalCore/tracing.hpp"

```

Comment:
Ah, clever. We don't register it in PublicTerminalCore, so VS doesn't have any logging pipelines to clean up. I think.

Reply:
Actually, this will become _very interesting_ when we merge the controls.. and further, when we offer them for 3p consumption.